buzzmobile/tests/test_utils/launch_tools.py

```diff
@@ -70,22 +77,23 @@ def launcher(func):
         def new_test(self):
             """Wrapper around the user-provided test that runs a ros node.
             """
-            node = roslaunch.core.Node(package, name, namespace=namespace)
+            env = {'ROS_MASTER_URI': self.rosmaster_uri}
+            node = roslaunch.core.Node(package, name, namespace=namespace,
+                    env_args=env.iteritems())
```

Comment:
Oh ok. I also misread `env_args` as `cmd_args` this morning when I was reviewing